internal/statusapi/statusapi.go

```diff
@@ -0,0 +1,107 @@
+package statusapi
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/clients"
+	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/configuration"
+	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/proxy"
+	"github.com/julienschmidt/httprouter"
+	log "github.com/sirupsen/logrus"
+)
+
+//JenkinsStatusAPI contains API to check whether Jenkins for the current user is idle|running|starting
+type JenkinsStatusAPI interface {
+	Status(w http.ResponseWriter, r *http.Request, ps httprouter.Params)
```

Comment:
we don't need a ps param here since it is unused.

Reply:
ps.ByName was giving errors. Getting rid of ps

Reply:
oops can't get rid of ps even if I don't need it. Otherwise router won't accept it as handle. Must be due to some httprouter requirement

Reply:
right, isn't golang convention to use `_` in that case?

Reply:
Done